lib/include/elements/support/font.hpp

```diff
@@ -308,6 +309,8 @@ namespace cycfi { namespace elements
 #ifdef __APPLE__
    fs::path get_user_fonts_directory();
 #endif
+
+   extern std::vector<fs::path> font_paths;
```

Comment:
I think it's better to make this a function that returns a "lazily" instantiated font_paths to avoid potential static initialization order issues.

Reply:
Agreed, do you think this solution to be appropriate?
46596836088e67c7dc23860a15a7c1bf92492a1c

Reply:
Looks good to me